webcompat/static/css/src/label-editor.css

```diff
@@ -70,8 +85,15 @@
 
 /* checkbox */
 .label-editor-list-item-checkbox {
-  margin: 0;
-  padding: 0;
+  display: none;
```

Comment:
I wonder if this would be better for accessibility @patrickhlauke 

```css
.label-editor-list-item-checkbox {
	/* display: none; */
	opacity: 0;
	order: 1000;
}
```
And do we need to give an outline on focus to the label?

Reply:
I personally don't remove the outline on focus, because it drastically reduces the ability for keyboard navigation.
For the input we can also use the `sr-only` helper.

Reply:
i'd need to see this in action and in context somewhere, but from memory (not tested in a while) `opacity:0` was a bit inconsistent on whether or not browser/AT combinations still announced it or not. If the intent here is to have something visually hidden, but announced by AT, CSS off-screen techniques are still the most reliable.

and yes, in cases where things like checkboxes/radio buttons are "faked" (assuming that's the context here?) it's not enough to not remove outline...as the thing that receives focus (the native checkbox/radio) is not on the screen/visible, there needs to be some explicit other way that shows/hides focus on the "faked" bit. randomly, i did some work ages ago on http://wtfforms.com/ which may show some of this approach - hoping it makes sense in this context